packages/components/src/Progress/Progress.component.js

```diff
@@ -40,8 +40,23 @@ function Progress({ id, percent, tooltip, infinite, contained }) {
 		);
 	}
 
+	let a11yProps;
+	if (!infinite) {
+		a11yProps = {
+			role: 'progressbar',
+			'aria-valuenow': normalizedPercent,
+			'aria-valuemin': 0,
+			'aria-valuemax': 100,
+		};
+	} else {
+		a11yProps = {
+			role: 'status',
+			'aria-busy': true,
+		};
+	}
+
 	return (
-		<div id={id} className={rootClassNames}>
+		<div id={id} className={rootClassNames} aria-label={tooltip} {...a11yProps}>
```

Comment:
why not adding aria-label in a11yProps ?

Reply:
Because I would have to put it twice, of to add some uglier code to insert it :)
In fact only the aria-label is constant here. The other a11 attributes are conditioned.